_pages/appendix-skyve-application-configuration.md

```diff
@@ -59,6 +64,32 @@ Alternatively, you can place the API key in the project JSON file under `api` ->
 	api: {
 		googleRecaptchaSiteKey: "xxxxxxxxxxx",
 ```
+#### Cloudflare Turnstile
+
+First, sign up for a Cloudflare Turnstile key as follows:
+1. Visit [Cloudflare](https://dash.cloudflare.com/sign-up) and sign up
+2. Click Turnstile in the sidebar menu on the left
+3. Click the 'Add site' button to add a new site
+4. Enter a site name to identify what this key will be for (the name of your Application)
+5. Enter the domain name of your application e.g 'website.com' or 'company.website.com'
```

Comment:
Maybe say of your _Skyve_ application so it is clearer which domain they need? They mention the subdomain so i think we want that.

```
Enter the domain name of your Skyve application e.g 'app.website.com' or 'website.com'
```